src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -438,6 +546,35 @@ nlohmann::json SysInfo::getPorts() const
         }
     }
 
+    std::deque<int64_t> inodes;
+
+    for (const auto& port : ports)
+    {
+        try
+        {
+            inodes.push_back(port.at("inode"));
+        }
+        catch (...) {}
+    }
```

Comment:
Just a question: if we populate the `inodes` array at the same time we get each port (line 542), we could avoid iterating the whole `ports` array again. Do you think it worth it?